lib/platforms/browser.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var React = require('react');
+
+var CONTAINER_ID = 'fluxapp-container';
+
+function renderComponent(routeStore, Component, serverSideRendered, routeId) {
+  if (routeStore.state.current.id === routeId) {
+    React.render(React.createElement(Component, {
+      serverSideRendered : true
+    }), document.getElementById(CONTAINER_ID));
+  }
+}
+
+module.exports = function initBrowser(fluxApp) {
+  var router = require('fluxapp-router');
+  router.use(fluxApp);
+
+  var routeStore = router.getStore();
+
+  routeStore.addChangeListener(function(change) {
+    var current = routeStore.state.current;
+    var route = current.route;
+    var isInitialRender = routeStore.state.history.length === 1;
+    var Component = route.handler;
+
+    if (isInitialRender) {
+      var state = JSON.parse(document.getElementById(CONTAINER_ID)
```

Comment:
it's guaranteed it will not, because we control what's in the tag by the lib/platforms/node